pandas/tests/indexes/test_category.py

```diff
@@ -581,6 +581,15 @@ def test_is_monotonic(self, data, non_lexsorted_data):
         assert c.is_monotonic_increasing
         assert not c.is_monotonic_decreasing
 
+    @pytest.mark.parametrize('values, expected', [
+        ([1, 2, 3], True),
+        ([1, 3, 1], False),
+        (list('abc'), True),
+        (list('aba'), False)])
+    def test_is_unique(self, values, expected):
+        ci = CategoricalIndex(values)
+        assert ci.is_unique == expected
```

Comment:
Can you use `is` instead of `==`?  A little bit of paranoia on my end, but it'd guard against the unlikely event that `is_unique` mistakenly starts returning equivalent non-booleans (e.g. 0/1).

Reply:
Ok, I've changed that.